### PR TITLE
Update ACTIONS - UNREIMBURSED UNINSURED DOCS.vbs

### PR DESCRIPTION
**Pull request by gretchent74**
removed county specific document

### CODE REVIEW

File: Script Files/ACTIONS/ACTIONS - UNREIMBURSED UNINSURED DOCS.vbs [hunk 1]
```diff
@@ -191,6 +228,133 @@ MsgBox ( "IMPORTANT!!  IMPORTANT!!" & vbNewline & vbNewline & "First select the
...
+	EMWriteScreen "s", 8, 5
+	transmit
+	EMWriteScreen "Y", 16, 15
+	transmit
+	PF3
+	EMWriteScreen "M", 3, 29
+	transmit
+
+'''need to select legal heading
+MsgBox ( "IMPORTANT!!  IMPORTANT!!" & vbNewline & vbNewline & "First select the correct LEGAL HEADING and press enter, " & vbNewline & "then PRESS OK so script can continue." )
```
Comment (RobertFewins-Kalb) [inline Script Files/ACTIONS/ACTIONS - UNREIMBURSED UNINSURED DOCS.vbs]: @gretchent74 there's a great msgbox function called vbSystemModal that keeps the msgbox in front of all open windows and prevents users from losing it behind their BlueZone session. Not required, just something to keep in mind.

File: Script Files/ACTIONS/ACTIONS - UNREIMBURSED UNINSURED DOCS.vbs [hunk 2]
```diff
@@ -191,6 +228,133 @@ MsgBox ( "IMPORTANT!!  IMPORTANT!!" & vbNewline & vbNewline & "First select the
...
+	EMWriteScreen "A", 3, 29
+	EMWriteScreen "F0949", 6, 36
+	EMWriteScreen "ncp", 11, 51
+	transmit
+	PF14
+	PF8
+	PF8	
+
+	EMWriteScreen "S", 11, 5
+	EMSendKey "<enter>"
```
Comment (RobertFewins-Kalb) [inline Script Files/ACTIONS/ACTIONS - UNREIMBURSED UNINSURED DOCS.vbs]: @gretchent74 can you please replace these EMSendKey "<enter>" with a transmit. The function includes an EMWaitReady to help slow the script down, should PRISM not be ready for the next input.

File: Script Files/ACTIONS/ACTIONS - UNREIMBURSED UNINSURED DOCS.vbs (hunk 1 above)
Comment (gretchent74) [inline Script Files/ACTIONS/ACTIONS - UNREIMBURSED UNINSURED DOCS.vbs]: Is there a sample on how to use it on different script.  I have been trying and can’t get it to work correctly.  Thanks.

Comment (theVKC) [inline Script Files/ACTIONS/ACTIONS - UNREIMBURSED UNINSURED DOCS.vbs]: I know we have something like that that Robert built into one of our MAXIS scripts: DAIL - ABAWD FSET EXEMPTION CHECK.vbs:

```vb.net
'Displaying the results...now with added MsgBox bling.
'vbSystemModal will keep the results in the foreground.
MsgBox closing_message, vbInformation + vbSystemModal, "ABAWD/FSET Exemption Check -- Results"
```

The constant `vbInformation` makes it look like an informative message (as opposed to a critical message, an error, etc) and the `vbSystemModal` turns it into a box that cannot be placed in the background (which strongly encourages your users to actually read and click it).

**Comment by gretchent74** [comment]
#325

File: Script Files/ACTIONS/ACTIONS - UNREIMBURSED UNINSURED DOCS.vbs (hunk 1 above)
Comment (gretchent74) [inline Script Files/ACTIONS/ACTIONS - UNREIMBURSED UNINSURED DOCS.vbs]: Thanks!

**Pull request merged**